examples/DynaSwing/IEC/Wind/Neplan/WPP4ACurrentSource/WPP4ACurrentSource.par

```diff
@@ -69,7 +69,7 @@
     <par type="DOUBLE" name="WPP_Kpufrt" value="2"/>
     <par type="DOUBLE" name="WPP_Kqv" value="2"/>
     <par type="INT" name="WPP_Mqfrt" value="1"/>
-    <par type="INT" name="WPP_MqG" value="1"/>
+    <par type="INT" name="WPP_MqG" value="2"/>
```

Comment:
I suggest that you add several examples for the different configurations of the table

Reply:
How would you do that?